README.md

```diff
@@ -79,7 +79,15 @@ This integration supports **Xiaomi MiBeacon, Qingping, ATC, Xiaomi Scale, Kegtro
 |**Ruuvitag**|**Ruuvitag**<br /><br />Round body. Broadcasts temperature, humidity, air pressure, battery voltage, battery level, motion and acceleration. If some of these sensors are not updating, make sure you use the latest firmware (v5). `motion detected` is reported in HA when the motion counter is increased between two advertisements. You can use the [reset_timer](#reset_timer) option to set the time after which the motion sensor will return to `motion clear`, but it might be overruled by the advertisements from the sensor.|![ruuvitag](/pictures/ruuvitag.jpg)|
 |**iNode Energy Meter**|**iNode Energy Meter**<br /><br />Energy meter based on pulse measuring. Broadcasts energy, power, battery and voltage. Energy and power are calculated based on the formula's as given in the [documentation](https://docs.google.com/document/d/1hcBpZ1RSgHRL6wu4SlTq2bvtKSL5_sFjXMu_HRyWZiQ/edit#heading=h.l38j4be9ejx7). The `constant` factor that is used for these calculations as well as the light level are given in the energy sensor attributes. Advertisements are broadcasted every 1 a 2 seconds, but the measurement data is only changed once a minute.|![iNode_Energy_Meter](/pictures/iNode_Energy_Meter.png)|
-  
+| **H5051** |**Govee H5051 Thermometer Hygrometer (BLE only)**<br/><br/>Oval body, LCD screen.  Broadcasts temperature, humidity and battery level.  Govee devices only send data with the option [active_scan](#active_scan) set to `True`, so make sure you change this setting, as the default is `False`|![H5051](/pictures/Govee_H5051.jpg)|
+| **H5072** |**Govee H5075 Thermometer Hygrometer**<br/><br/>Oval body, LCD screen.  Broadcasts temperature, humidity and battery level.  Govee devices only send data with the option [active_scan](#active_scan) set to `True`, so make sure you change this setting, as the default is `False`|![H5072](/pictures/Govee_H5072.jpg)|
+| **H5074** |**Govee H5074 Thermometer Hygrometer**<br/><br/>Square body, no screen.  Broadcasts temperature, humidity and battery level.  Govee devices only send data with the option [active_scan](#active_scan) set to `True`, so make sure you change this setting, as the default is `False`|![H5074](/pictures/Govee_H5074.jpg)|
+| **H5075** |**Govee H5075 Thermometer Hygrometer**<br/><br/>Rounded square body, LCD screen.  Broadcasts temperature, humidity and battery level.  Govee devices only send data with the option [active_scan](#active_scan) set to `True`, so make sure you change this setting, as the default is `False`|![H5075](/pictures/Govee_H5075.jpg)|
+| **H5101, H5102** |**Govee H5101/Govee H5102 Thermometer Hygrometer**<br/><br/>Rounded square body, LCD screen.  Broadcasts temperature, humidity and battery level.|![H5101/H5102](/pictures/Govee_H5101_H5102.jpg)|
```

Comment:
No active scan for H5101/H5102?

Reply:
It is required.  I'm updating it.  Thanks for catching that.